recipes/radical.utils/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "radical.utils" %}
+{% set version = "0.47.5" %}
+{% set sha256 = "423c1959c7b387b0a54128a7263d46c6673a247fd4e06de4e728b7c81005d109" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/radical-cybertools/radical.utils/
```

Comment:
Can you change this to point to an archive tag instead of a git url?  This makes the recipe substantially more reproducible.  Or even better a package on pypi

Reply:
@iparask: pypi package, i.e. the released version, would make sense?

Reply:
Yes, it does.